src/QuickMarcEditor/QuickMarcCreateBibWrapper.test.js

```diff
@@ -23,47 +23,115 @@ jest.mock('react-final-form', () => ({
 }));
 
 const mockFormValues = jest.fn(() => ({
-  fields: undefined,
-  externalHrid: 'in00000000022',
-  externalId: '17064f9d-0362-468d-8317-5984b7efd1b5',
-  leader: '00000nu\\\\\\2200000un\\4500',
-  marcFormat: 'HOLDINGS',
-  parsedRecordDtoId: '1bf159d9-4da8-4c3f-9aac-c83e68356bbf',
-  parsedRecordId: '1bf159d9-4da8-4c3f-9aac-c83e68356bbf',
-  records: [
+  'externalId': '00000000-0000-0000-0000-000000000000',
```

Comment:
Usually we don't use quotes for object keys. Please update it to follow current pattern